HomePage/HomePage.es.json

```diff
@@ -1,9 +1,9 @@
 {
-  "About": "Información",
+  "About": "Info",
   "AddCustomSensor": "Añadir \"{0}\" sensor personalizado",
   "AddFanCurve": "Añadir \"{0}\" curva",
```

Comment:
Generally the "..." are a sign that a new dialog will result from that action, so you don't want to remove it generally.

Reply:
Added on last commit.